composer.json

```diff
@@ -23,7 +23,7 @@
         "ext-exif": "*",
         "ext-fileinfo": "*",
         "ext-gd": "*",
-        "api-platform/core": "^2.5",
+        "api-platform/core": "2.5.*",
```

Comment:
You should rebase with master and this change should be reverted as there was added a conflict to `"api-platform/core": "^2.6"`